src/main.rs

```diff
@@ -228,13 +231,31 @@ fn main() {
 /// Write Dignostics to STDERR
 ///
 fn dump_diagnostics(source: &text::SourceText, diagnostics: &[diag::Diagnostic]) {
+    let mut pad = false;
     for error in diagnostics.iter() {
-        // TODO: Move this formatting into some library code.
+        if pad {
+            eprintln!("");
```

Comment:
If this padding is moved after each error it can be come unconditional. Then we remove the padding before the error summary line.